control-plane/plugin/src/resources/mod.rs

```diff
@@ -46,6 +48,13 @@ pub enum ScaleResources {
     },
 }
 
+/// The types of resources that support cordoning.
+#[derive(clap::Subcommand, Debug)]
+pub enum CordonResources {
+    /// Cordon the node with the given ID.
```

Comment:
Should we also add a word or two about the `labels` as this probably would become the document in the `--help`.

Reply:
Done.